charts/immich/templates/server/deployment.yaml

```diff
@@ -3,40 +3,31 @@ apiVersion: apps/v1
 metadata:
   name: {{ template "common.fullname" . }}-server
   labels:
-    app.kubernetes.io/name: {{ template "common.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/component: "immich-server"
+    app.kubernetes.io/component: "server"
     {{- include "common.labels" . | nindent 4 }}
 spec:
   revisionHistoryLimit: 0
   replicas: {{ .Values.server.replicas }}
-  strategy:
-    type: Recreate
```

Comment:
Why did we use recreate? Immich says they support multiple replicas. Does this not work properly?

Reply:
Didn't expect them to support multiple replicas and didn't look into it tbh 👍🏼